.github/workflows/preview-listen-fe.yml

```
@@ -137,10 +137,44 @@ jobs:
+        uses: treosh/lighthouse-ci-action@v12
+        with:
+          urls: |
+            ${{ steps.deploy.outputs.details_url }}
+          # budgetPath: apps/listen/budget.json # test performance budgets
+          configPath: apps/listen/lighthouserc.json
+          uploadArtifacts: true # save results as an action artifacts
+          temporaryPublicStorage: true # upload lighthouse report to the temporary storage
+      - name: Comment PR
+        uses: thollander/actions-comment-pull-request@v3
+        with:
+          message: |
```

actionlint (1.7.4)

property "manifest" is not defined in object type {assertionresults: string; links: string; resultspath: string}

(expression)